plugins/email_service/app/helpers/email_service/application_helper.rb

```diff
@@ -1093,10 +1380,20 @@ def find_custom_verification_email_template(name)
 
     ### nebula
 
-    ## 
+    def custom_endpoint_url
+      return nil
+    end
+
+    def nebula_provider_collection
+      ["aws", "int"]
+    end
+
+    def nebula_endpoint_url
+      @nebula_endpoint = current_user.service_url('nebula')
+    end
 
     def _nebula_request(uri, method, headers = nil, body = nil )
-      # puts "Parsing URL: #{uri}"
+
       https = Net::HTTP.new(uri.host, uri.port)
```

Comment:
Please switch all your own client request to use elektron.

Reply:
removed.